pytensor/utils.py

```diff
@@ -111,6 +111,17 @@ def cmp(x, y):
     return (x > y) - (x < y)
 
 
+def key_to_cmp(key):
```

Comment:
Was this used anywhere else? I saw it was introduced in the same PR as the MIP

Reply:
This function was used only once for the sched module

Reply:
Can we remove it then?

Reply:
just did